src/collections/componentSet.ts

```diff
@@ -523,6 +523,14 @@ export class ComponentSet extends LazyCollection<MetadataComponent> {
         if (parentInWildcard) {
           return true;
         }
+        const partialWildcardKey = this.simpleKey({
```

Comment:
I added support for this. Incase not supporting this was WAD, do let me know, I'll remove this code from this PR

Reply:
`What would be the difference between asking for site/Xcel_Energy1 of a DEB and site/Xcel_Energy1.* of a DE?`
We won't retrieve -meta.xml of the DEB. You are right that expect this there won't be any other changes, it will only be helpful if there are multiple child types for a parent md type. For DEB there is only one child type for now, so I'm okay to not support this as well